README.md

```diff
@@ -1880,16 +1876,17 @@ Any good resources on this?
 
 ## Taxes on Equity Compensation
 
-Now that we’ve covered the basic concepts of equity and taxes, we’ll get into some messy
-details of how they interact.
-
-As already discussed, employees can get restricted stock, stock options, or RSUs.
-The tax consequences for each of these is dramatically different.
+It is crucial to understand the tax implications of stock options and awards.
+Equity and taxes interact in complicated ways, and the tax consequences for an employee’s
+restricted stock, stock options, or RSUs are dramatically different.
+This section will cover these messy details and help you avoid outsized taxes on your
```

Comment:
"avoid outsized ..." -> "plan and make decisions that reduce the tax burden of your equity compensation"

Reply:
(often you can't avoid taxes, but you can plan for them)

Reply:
fixed

Reply:
(cut "plan and")